### PR TITLE
Create pug file for Alexandra's columns page

### PR DESCRIPTION
**Pull request by Legai**
Necessary css files was put to the appropriate directories
Necessary adv002Controller.js, allRenderData.js files were created
router.js was modified

### CODE REVIEW

File: backend/public/css/col002CommonStyle.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.title {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 600;
+}
+
+.describe {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 300;
+}
```
Comment (102) [inline backend/public/css/col002CommonStyle.css]: Not this time

File: backend/public/css/col002CommonStyle.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        max-width: 960px;
+    }
+}
+
+.title {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 600;
+}
+
+.describe {
```
Comment (mvahrameev) [inline backend/public/css/col002CommonStyle.css]: I can't find where this class is used

File: backend/app/controllers/col002Controller.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+var allData = require('../../dataForRender/allRenderData');
+
+module.exports = (request, response) => {
+    response.render('pages/col002Render', {
+        divParams: allData.divParamsForCol002,
+        rowText: allData.rowTextForCol002,
+    });
+};
```
Comment (mvahrameev) [inline backend/app/controllers/col002Controller.js]: Use more sensible names for variables, controllers, views and etc. `Col002` is quite strange name

File: backend/app/views/pages/col002Render.pug [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+doctype html
+html
+head
+    meta(charset="UTF-8")
+    meta(name="viewport" content="width=device-width,initial-scale=1")
+    link(rel="stylesheet" href="css/col002CommonStyle.css")
+    link(rel="stylesheet" href="css/col002Style.css")
+body
+    div(class=divParams[0])
+        div(class=divParams[1])
```
Comment (mvahrameev) [inline backend/app/views/pages/col002Render.pug]: Why do you need template variables for class names?

File: backend/public/css/col002Style.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        font-size: 14px;
+        line-height: 1.45;
+    }
+}
+
+tr:nth-child(even) {
+    background-color: #eeeeee;
+    border-top: 1px solid #d9d9d9;
+    border-bottom: 1px solid #d9d9d9;
+
```
Comment (IgorZiegler) [inline backend/public/css/col002Style.css]: Remove empty lines

File: .gitignore [hunk 6]
```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+.idea/*
+.idea/workspace.xml
```
Comment (102) [inline .gitignore]: This line is redundant

**Pull request closed without merging**